lib/api/BackbeatAPI.js

```diff
@@ -309,6 +309,38 @@ class BackbeatAPI {
         ], cb);
     }
 
+    /**
+     * Delete the failed CRR Redis hash field.
+     * @param {String} field - The field in the hash to delete
+     * @param {Function} cb - The callback function
+     * @return {undefined}
+     */
+    _deleteFailedCRRField(field, cb) {
+        const cmds = ['hdel', redisKeys.failedCRR, field];
+        return this._redisClient.batch([cmds], (err, res) => {
```

Comment:
just making sure, using the square brackets around `cmds` will make this a 2d array